lepidopter-vmdebootstrap_build.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -exa
+GIT_BUILD="$(git describe)"
```

Comment:
I get the same output with `git describe` and `git describe --tags`.
Why should be a difference between release and non release ? 
Does `git describe` not always providing the last generated commit hash, or am I missing anything?